scripts/changed-modules.sh

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
```

Comment:
suggestion: set the options to detect issues e.g `set -euxo pipefail` if you want to be really strict.